pkg/chains/objects/objects.go

```diff
@@ -107,6 +108,20 @@ func (tro *TaskRunObject) GetServiceAccountName() string {
 	return tro.Spec.ServiceAccountName
 }
 
+// GetParent returns the PipelineRun object for the TaskRun, if one exists.
+func (tro *TaskRunObject) GetParent(ctx context.Context, clientSet versioned.Interface) (TektonObject, error) {
+	prName, ok := tro.GetLabels()["tekton.dev/pipelineRun"]
```

Comment:
nit: https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#OwnerReference is the more - authoritative source for this.

Reply:
Thanks for the suggestion. Updated.